accepted/0000-link-packages-to-source-and-build.md

```diff
@@ -0,0 +1,514 @@
+# Link npm packages to the originating source code repository and build
```

Comment:
[View rendered version](https://github.com/npm/rfcs/blob/link-packages-to-source-and-build/accepted/0000-link-packages-to-source-and-build.md)